### ISSUE TITLE
Include SVG 1.1

### ISSUE THREAD

**Issue by pyoor** [w3c/browser-specs#273]
Currently, only the SVG2 (and related) specs are included.  However, SVG2 only includes additions or changes to the SVG 1 specification.  For completeness, it would be useful to include the SVG1.1 spec as well.

Note, some of these values may be incorrect.  I based these entries on the SVG2 spec definition.

```javascript
{
    'url': 'https://www.w3.org/TR/SVG11/',
    'seriesComposition': 'full',
    'shortname': 'SVG11',
    'series': {
      'shortname': 'SVG11',
      'currentSpecification': 'SVG11'
    },
    'seriesVersion': '1',
    'release': {
      'url': 'https://www.w3.org/TR/SVG11/',
      'pages': [
        'https://www.w3.org/TR/SVG11/access.html',
        'https://www.w3.org/TR/SVG11/animate.html',
        'https://www.w3.org/TR/SVG11/attindex.html',
        'https://www.w3.org/TR/SVG11/backward.html',
        'https://www.w3.org/TR/SVG11/changes.html',
        'https://www.w3.org/TR/SVG11/color.html',
        'https://www.w3.org/TR/SVG11/concepts.html',
        'https://www.w3.org/TR/SVG11/conform.html',
        'https://www.w3.org/TR/SVG11/coords.html',
        'https://www.w3.org/TR/SVG11/eltindex.html',
        'https://www.w3.org/TR/SVG11/escript.html',
        'https://www.w3.org/TR/SVG11/expanded-toc.html',
        'https://www.w3.org/TR/SVG11/extend.html',
        'https://www.w3.org/TR/SVG11/feature.html',
        'https://www.w3.org/TR/SVG11/filters.html',
        'https://www.w3.org/TR/SVG11/fonts.html',
        'https://www.w3.org/TR/SVG11/i18n.html',
        'https://www.w3.org/TR/SVG11/idl.html',
        'https://www.w3.org/TR/SVG11/implnote.html',
        'https://www.w3.org/TR/SVG11/index.html',
        'https://www.w3.org/TR/SVG11/interact.html',
        'https://www.w3.org/TR/SVG11/intro.html',
        'https://www.w3.org/TR/SVG11/java.html',
        'https://www.w3.org/TR/SVG11/linking.html',
        'https://www.w3.org/TR/SVG11/masking.html',
        'https://www.w3.org/TR/SVG11/metadata.html',
        'https://www.w3.org/TR/SVG11/mimereg.html',
        'https://www.w3.org/TR/SVG11/minimize.html',
        'https://www.w3.org/TR/SVG11/Overview.html',
        'https://www.w3.org/TR/SVG11/painting.html',
        'https://www.w3.org/TR/SVG11/paths.html',
        'https://www.w3.org/TR/SVG11/propidx.html',
        'https://www.w3.org/TR/SVG11/pservers.html',
        'https://www.w3.org/TR/SVG11/refs.html',
        'https://www.w3.org/TR/SVG11/render.html',
        'https://www.w3.org/TR/SVG11/script.html',
        'https://www.w3.org/TR/SVG11/shapes.html',
        'https://www.w3.org/TR/SVG11/single-page.html',
        'https://www.w3.org/TR/SVG11/struct.html',
        'https://www.w3.org/TR/SVG11/styling.html',
        'https://www.w3.org/TR/SVG11/svgdom.html',
        'https://www.w3.org/TR/SVG11/svgdtd.html',
        'https://www.w3.org/TR/SVG11/text.html',
        'https://www.w3.org/TR/SVG11/types.html',
      ],
      'filename': 'Overview.html'
    },
    'nightly': {
      'url': 'https://www.w3.org/TR/SVG11/',
      'pages': [
        'https://www.w3.org/TR/SVG11/access.html',
        'https://www.w3.org/TR/SVG11/animate.html',
        'https://www.w3.org/TR/SVG11/attindex.html',
        'https://www.w3.org/TR/SVG11/backward.html',
        'https://www.w3.org/TR/SVG11/changes.html',
        'https://www.w3.org/TR/SVG11/color.html',
        'https://www.w3.org/TR/SVG11/concepts.html',
        'https://www.w3.org/TR/SVG11/conform.html',
        'https://www.w3.org/TR/SVG11/coords.html',
        'https://www.w3.org/TR/SVG11/eltindex.html',
        'https://www.w3.org/TR/SVG11/escript.html',
        'https://www.w3.org/TR/SVG11/expanded-toc.html',
        'https://www.w3.org/TR/SVG11/extend.html',
        'https://www.w3.org/TR/SVG11/feature.html',
        'https://www.w3.org/TR/SVG11/filters.html',
        'https://www.w3.org/TR/SVG11/fonts.html',
        'https://www.w3.org/TR/SVG11/i18n.html',
        'https://www.w3.org/TR/SVG11/idl.html',
        'https://www.w3.org/TR/SVG11/implnote.html',
        'https://www.w3.org/TR/SVG11/index.html',
        'https://www.w3.org/TR/SVG11/interact.html',
        'https://www.w3.org/TR/SVG11/intro.html',
        'https://www.w3.org/TR/SVG11/java.html',
        'https://www.w3.org/TR/SVG11/linking.html',
        'https://www.w3.org/TR/SVG11/masking.html',
        'https://www.w3.org/TR/SVG11/metadata.html',
        'https://www.w3.org/TR/SVG11/mimereg.html',
        'https://www.w3.org/TR/SVG11/minimize.html',
        'https://www.w3.org/TR/SVG11/Overview.html',
        'https://www.w3.org/TR/SVG11/painting.html',
        'https://www.w3.org/TR/SVG11/paths.html',
        'https://www.w3.org/TR/SVG11/propidx.html',
        'https://www.w3.org/TR/SVG11/pservers.html',
        'https://www.w3.org/TR/SVG11/refs.html',
        'https://www.w3.org/TR/SVG11/render.html',
        'https://www.w3.org/TR/SVG11/script.html',
        'https://www.w3.org/TR/SVG11/shapes.html',
        'https://www.w3.org/TR/SVG11/single-page.html',
        'https://www.w3.org/TR/SVG11/struct.html',
        'https://www.w3.org/TR/SVG11/styling.html',
        'https://www.w3.org/TR/SVG11/svgdom.html',
        'https://www.w3.org/TR/SVG11/svgdtd.html',
        'https://www.w3.org/TR/SVG11/text.html',
        'https://www.w3.org/TR/SVG11/types.html',
      ],
      'filename': 'Overview.html'
    },
    'title': 'Scalable Vector Graphics (SVG) 1.1 (Second Edition)',
    'source': 'w3c',
    'shortTitle': 'SVG 11',
    'tests': {
      'repository': 'https://github.com/web-platform-tests/wpt',
      'testPaths': [
        'svg'
      ]
    }
  }
```

**Comment by tidoust** [comment]
For specs that have multiple versions, the approach we've taken so far to determine whether previous versions should be added to the list is to look at what the version-less shortname redirects to. In this case, https://www.w3.org/TR/SVG/ redirects to SVG 2, published as a Candidate Recommendation (which should convey some notion of stability). That spec presents itself as building "upon SVG 1.1 Second Edition by improving the usability and precision of the language". It should not only include additions or changes to SVG 1. What is missing exactly?

FYI, the bits you provided would be automatically generated by the generating code. All that is needed to add SVG 1.1 to the list is to add a line `https://www.w3.org/TR/SVG11/ multipage` to `specs.json` (see [contributing doc](https://github.com/w3c/browser-specs/blob/master/CONTRIBUTING.md#how-to-addupdatedelete-a-spec)). The generating code should take care of the rest.

**Comment by pyoor** [comment]
My apologies - by missing I meant the attributes that have been deprecated by SVG2.  My only argument for including SVG1 is that most browsers do not support SVG2 in full (AFAIK) and further that some of those deprecated features are still in use.  Though, I also understand and agree with your reasoning for only including SVG2.

**Comment by Elchi3** [comment]
MDN also documents a few SVG 1.1 only features. I haven't checked if the docs should rather be archived but given they currently exist, it would also be useful for us if browser-specs had SVG 1.1.

Here's what mdn-browser-compat-data references right now and I believe these things aren't in SVG2. Happy to discuss what to do about these.

 ```
 'https://www.w3.org/TR/SVG11/animate.html#AnimateColorElement',
  'https://www.w3.org/TR/SVG11/color.html#ColorProfileElement',
  'https://www.w3.org/TR/SVG11/color.html#ColorProfileProperty',
  'https://www.w3.org/TR/SVG11/filters.html#EnableBackgroundProperty',
  'https://www.w3.org/TR/SVG11/fonts.html#FontElement',
  'https://www.w3.org/TR/SVG11/fonts.html#FontFaceElement',
  'https://www.w3.org/TR/SVG11/fonts.html#FontFaceFormatElement',
  'https://www.w3.org/TR/SVG11/fonts.html#FontFaceNameElement',
  'https://www.w3.org/TR/SVG11/fonts.html#FontFaceSrcElement',
  'https://www.w3.org/TR/SVG11/fonts.html#FontFaceURIElement',
  'https://www.w3.org/TR/SVG11/fonts.html#GlyphElement',
  'https://www.w3.org/TR/SVG11/fonts.html#HKernElement',
  'https://www.w3.org/TR/SVG11/fonts.html#MissingGlyphElement',
  'https://www.w3.org/TR/SVG11/fonts.html#VKernElement',
  'https://www.w3.org/TR/SVG11/linking.html#XLinkArcRoleAttribute',
  'https://www.w3.org/TR/SVG11/linking.html#XLinkShowAttribute',
  'https://www.w3.org/TR/SVG11/linking.html#XLinkTypeAttribute',
  'https://www.w3.org/TR/SVG11/struct.html#ExternalResourcesRequiredAttribute',
  'https://www.w3.org/TR/SVG11/struct.html#RequiredFeaturesAttribute',
  'https://www.w3.org/TR/SVG11/struct.html#XMLBaseAttribute',
  'https://www.w3.org/TR/SVG11/text.html#AltGlyphDefElement',
  'https://www.w3.org/TR/SVG11/text.html#AltGlyphElement',
  'https://www.w3.org/TR/SVG11/text.html#AltGlyphItemElement',
  'https://www.w3.org/TR/SVG11/text.html#GlyphOrientationHorizontalProperty',
  'https://www.w3.org/TR/SVG11/text.html#GlyphOrientationVerticalProperty',
  'https://www.w3.org/TR/SVG11/text.html#GlyphRefElement',
  'https://www.w3.org/TR/SVG11/text.html#TRefElement',
```

**Comment by tidoust** [comment]
I added SVG 1.1 to browser-specs, released in [v1.35.0](https://github.com/w3c/browser-specs/releases/tag/1.35.0). There is no dedicated nightly document for SVG 1.1, so the `nightly` section actually targets the SVG 2 Editor's Draft.
